pkg/cmd/cluster/cluster.go

```diff
@@ -0,0 +1,31 @@
+package cluster
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	"github.com/buildkite/cli/v3/pkg/cmd/validation"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdCluster(f *factory.Factory) *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "cluster <command>",
+		Short: "View cluster information",
```

Comment:
This one isn't technically correct since this is the parent `cluster` command. Probably change it to the Long version

Reply:
Done